docs/updating/4_update_2.5.md

```diff
@@ -52,3 +52,24 @@ or by executing the following command:
 ```bash
 bin/console cache:pool:clear cache.redis
 ```
+
+## Updating to 2.5.3
+
+### Page builder
+
+!!! enterprise
+
+    This step is only **required when updating from versions higher than 2.2 and lower than 2.5.3**.
+    In case of versions lower than 2.2, please skip this step or ignore the information that  indexes from a script below already exist.
```

Comment:
```suggestion
    In case of versions lower than 2.2, please skip this step or ignore the information that indexes from a script below already exist.
```